apps/package.json

```diff
@@ -57,11 +57,9 @@
     "grunt-notify": "0.4.1",
     "grunt-sass": "1.0.0",
     "grunt-strip-code": "0.1.2",
-    "handlebars-runtime": "1.0.12",
```

Comment:
Not sure offhand. Could probably figure this out by git blaming, but I assume you'd need to go into our old repo to do that.